brave/browser-laptop
# Browser-laptop changes for trigger ask user consent before autofill

fix #6558

Auditors: @bridiver, @diracdeltas, @bbondy

Test Plan:
1. Make sure you have autofill profile in about:autofill
2. Go to https://anttiviljami.github.io/browser-autofill-phishing/
3. Click name field to trigger autofill
4. Select one suggestion
5. There will be a dialog informs users the values about to auotfill
6. If user click ok, the values will be filled.
   If user click cancel, no values will be filled.

Submitter Checklist:


Test Plan:


Reviewer Checklist:


Tests
